src/styles/__mocks__/styled.js

```diff
@@ -84,6 +84,7 @@ export function styled(
           ref={this.props.forwardedRef}
           styled-component="true"
           test-style={this.state.styles}
+          test-style-stringified={JSON.stringify(this.state.styles)}
```

Comment:
Hm, do we really want to add this?  It creates a lot of noise. At least it should be formatted into multiple lines so future diffs are reviewable.

Reply:
I can format it, but it with react-testing-lib getting the attr `test-style` will only say `[object: Object]`. Do you have any ideas for how to test?

Reply:
I'm going to replace our mock with https://github.com/styletron/styletron/tree/master/packages/styletron-engine-snapshot